core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -414,7 +416,14 @@ export class RteHelperModalComponent {
             this.tmpCustomizationArgs[i].value.toString()
           );
         }
+        (
```

Comment:
I don't quite understand what's going on here ... can we find a way to avoid repeating the same logic in every branch of the code?

Reply:
I changed the save() function to be divided in an if clause. Basically the process is very similar for all components that aren't mathExpression so I changed it to a way that I don't repeat the same logic. 

Done, PTAL to my new changes @seanlip